CHANGELOG.md

```diff
@@ -58,11 +58,13 @@ should change the heading of the (upcoming) version to include a major version b
 
 ## @rjsf/shadcn
 
+- Update `README.md` with picture of the theme!
```

Comment:
Can you create a new `beta.15` release section with these changes since we release `beta.14` already